forge-gui/res/tokenscripts/c_1_1_insect_artifact_flying.txt

```diff
@@ -0,0 +1,6 @@
+Name:Insect Token
```

Comment:
File name should be c_1_1_a_insect_flying.txt

and the script should reference this too